docs/odk-workflows/RepositoryFileStructure.md

```diff
@@ -34,6 +34,7 @@ These are the components in MONDO-INGEST
 | gard.owl | https://github.com/monarch-initiative/gard/releases/latest/download/gard.owl |
 | icd10cm.owl | https://data.bioontology.org/ontologies/ICD10CM/submissions/23/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb |
 | icd10who.owl | https://github.com/monarch-initiative/icd10who/releases/latest/download/icd10who.ttl |
+| icd11.owl | https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz |
```

Comment:
## Docs
_[Nico wrote](https://github.com/monarch-initiative/mondo-ingest/pull/412#discussion_r1485633597)_:
> add what you know and we will iterate (where we got the file from, for instance, and maybe a link to an external page about what the ICD11 foundation is)
- [ ] @joeflack4 By the time I've undrafted PR, add anything I can think of to `docs/sources/icd11.md`
- [ ] @joeflack4 Perhaps also add: where we got the file, link to what ICD11 foundation is

Reply:
@matentzn @twhetzel I just added the following.

> **Comments about this source:**  
> _Data source_
> _Original source URL_: https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz
> 
> _Preprocessing_
> In the [monarch-initiative/icd11](https://github.com/monarch-initiative/icd11) repo, We remove unicode characters and then remove equivalent class statements as discussed below. 
> 
> _Equivalent classes_  
> We remove all equivalent class statements as they are not unique and result in unintended node merges. For example  `icd11.foundation:2000662282` (_Occupant of pick-up truck or van injured in collision with car, pick-up truck or van: person on outside of vehicle injured in traffic accident_) has the same exact equivalent concept expression as `icd11.foundation:1279712844` (_Occupant of pick-up truck or van injured in collision with two- or three- wheeled motor vehicle: person on outside of vehicle injured in traffic accident_).

Marking resolved.